webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -110,6 +220,55 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           ]}>
           <Input maxLength={80} />
         </Form.Item>
+        {canModifyBindings && Loadable.getOrElse([], resourceManagers).length > 0 && (
+          <>
+            <Divider />
+            Namespace Bindings
+            <Body inactive>
+              Note: If you leave the Namespace name blank, the workspace will be bound to the
+              default Namespace configured in the Master Config.
+            </Body>
+            <>
+              {Loadable.getOrElse([], resourceManagers).map((name) => (
+                <Fragment key={name}>
+                  <Form.Item label={name} name={['bindings', name, 'namespace']}>
+                    <Input
+                      disabled={watchBindings?.[name]?.['autoCreateNamespace'] ?? false}
+                      maxLength={63}
+                    />
+                  </Form.Item>
+                  {info.branding === BrandingType.HPE && (
+                    <>
+                      <Form.Item
+                        label="Auto Create Namespace"
+                        name={['bindings', name, 'autoCreateNamespace']}
+                        valuePropName="checked">
+                        <Toggle
+                          onChange={() => form.setFieldValue(['resourceQuotas', name], undefined)}
```

Comment:
this doesn't make sense:
the `Form.Item` is hooked up to `bindings.[name].autoCreateNamespace`, but the toggle sets `resourceQuotas.[name]` to undefined on every change. What was the intent here?

Reply:
This LOC isn't part of this PR, the feature branch https://github.com/determined-ai/determined/pull/9180 got rebased onto main, changing the commit history. 

cc @salonig23 if this is a webUI change needed

Reply:
In the Create Workspace scenario - 
They are only able to edit the resource quota if auto-create-namespace toggle is on. So if they turn it on and we reset it to undefined it doesn't matter because it should be undefined to begin with. If they input something and turn the toggle off, it should reset it to undefined because they cannot set resource quota unless the toggle is on.

In the Edit workspace scenario:
The resource quota field auto populates with the resource quota for that namespace from the kubernetes informer. This field is greyed out when the namespace is not auto-created, and editable when it was autocreated. 
- If the namespace wasn't autocreated, when they open the edit modal the autocreate toggle will be off. if they decide to turn it on, then we reset the field back to undefined so it is a clean slate for the new namespace.
- If the namespace was autocreated, the toggle should be on when they open the edit modal. If they decide to change the toggle to off, then we reset the field back to undefined because that previously auto-poulated quota doesn't apply to the namespace they will be inputting and this field is not editable when toggle is off anyway

Reply:
@ashtonG does that clarify the reasoning?

Reply:
Reiterating that this code is from a separate PR, so I'm resolving this conversation here